packages/cactus-plugin-bungee-hermes/src/main/typescript/view-creation/proof.ts

```diff
@@ -0,0 +1,48 @@
+export class Proof {
+  private creator: string;
+  private mspid?: string;
+  private signature?: string;
+
+  constructor(settings: {
+    creator: string;
+    mspid?: string;
+    signature?: string;
+  }) {
+    this.creator = settings.creator;
+    this.mspid = settings.mspid ? settings.mspid : "undefined";
+    this.signature = settings.signature ? settings.signature : "undefined";
+  }
+
+  public printEndorsement(): string {
+    return (
+      "Endorsement: \n " +
+      this.creator +
+      " \n " +
+      this.mspid +
+      " \n " +
+      this.signature
+    );
+  }
+
+  public getEndorsementJson(): string {
+    const proof = {
+      creator: this.creator,
+      mspid: this.mspid,
+      signature: this.signature,
+    };
+    // JSON.stringify(Array.from(endors.entries()));
+    return JSON.stringify(proof);
+  }
+
+  public getCreator(): string {
```

Comment:
Can we add a quick in-code comment here describing the nature of creator for the ledgers? It would be very helpful I believe, something like

The term creator refers to `endorsementID` for fabric ledger or `txTx.data.transaction.from` in case of besu ledger

Reply:
+1